### ISSUE TITLE
test_column_different_sizes fails with Omniscidb 5.10 and CUDA enable

### ISSUE THREAD

**Issue by guilhermeleobas** [xnd-project/rbc#408]
Test `test_omnisci_column_basic.py::test_column_different_sizes` fails with OmniSciDB 5.10 and CUDA enabled.

```
rbc/tests/test_omnisci_column_basic.py::test_column_different_sizes FAILED                                                                                                                                [100%]

=================================================================================================== FAILURES ====================================================================================================
__________________________________________________________________________________________ test_column_different_sizes __________________________________________________________________________________________
rbc/tests/test_omnisci_column_basic.py:822: in test_column_different_sizes
    assert list(result) == expected
E   assert [(80.0,), (48...,), (4650.0,)] == [(10.0,), (60...0,), (300.0,)]
E     At index 0 diff: (80.0,) != (10.0,)
E     Full diff:
E     - [(10.0,), (60.0,), (180.0,), (240.0,), (300.0,)]
E     + [(80.0,), (480.0,), (1140.0,), (2040.0,), (4650.0,)]
```

**Comment by pearu** [comment]
This issue was fixed in https://github.com/omnisci/omniscidb-internal/pull/6187 but it looks like it did not land in 5.10.

For omniscidb 5.10 conda forge package, we'll need to patch the feedstock accordingly.

**Comment by pearu** [comment]
omniscidb 5.10.1 fixes the issue.

**Comment by guilhermeleobas** [comment]
Closing this PR as it was fixed in OmniSciDB 5.10
